config/storage.py

```diff
@@ -1,84 +1,79 @@
-""" Storage Settings """
+"""Storage Settings."""
 
-import os
+from masonite import env
 
-"""
-|--------------------------------------------------------------------------
-| Storage Driver
-|--------------------------------------------------------------------------
-|
-| The default driver you will like to use for storing uploads. You may add
-| additional drivers as you need or pip install additional drivers.
-|
-| Supported: 'disk', 's3'
-|
+"""Storage Driver
+The default driver you will like to use for storing uploads. You may add
+additional drivers as you need or pip install additional drivers.
+
+Supported: 'disk', 's3', 'rackspace', 'googlecloud', 'azure'
 """
 
-DRIVER = os.getenv('STORAGE_DRIVER', 'disk')
+DRIVER = env('STORAGE_DRIVER', 'disk')
 
-"""
-|--------------------------------------------------------------------------
-| Storage Drivers
-|--------------------------------------------------------------------------
-|
-| Different drivers you can use for storing file uploads.
-|
+"""Storage Drivers
+Different drivers you can use for storing file uploads.
 """
 
 DRIVERS = {
     'disk': {
-        'location': {
-            'uploading': 'uploads/'
-        }
+        'location': 'storage/uploads'
     },
     's3': {
-        'client': os.getenv('S3_CLIENT', 'AxJz...'),
-        'secret': os.getenv('S3_SECRET', 'HkZj...'),
-        'bucket': os.getenv('S3_BUCKET', 's3bucket'),
-        'location': 'http://s3.amazon.com/bucket',
-    }
+        'client': env('S3_CLIENT', 'AxJz...'),
+        'secret': env('S3_SECRET', 'HkZj...'),
+        'bucket': env('S3_BUCKET', 's3bucket'),
```

Comment:
The reason the tests are failing is because you are missing the `location` here

Reply:
thanks for pointing out